modules/apps/commerce/commerce-theme-minium/commerce-theme-minium-site-initializer/src/main/resources/com/liferay/commerce/theme/minium/site/initializer/internal/dependencies/layouts.json

```diff
@@ -249,6 +249,27 @@
 		],
 		"privateLayout": false
 	},
+	{
+		"icon": "icon_orders.png",
+		"layoutTemplateId": "1_column",
+		"name": "Returns",
+		"permissions": [
+			{
+				"actionIds": [
+				],
+				"roleName": "Guest",
```

Comment:
Guest will never be able to place returns I guess, does it makes sense to make the page available to it?

Reply:
Removed, also as per https://liferay.slack.com/archives/G9CMBBA9H/p1711370833030989?thread_ts=1711365042.099939&cid=G9CMBBA9H 👍🏼